specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -154,6 +163,33 @@ export function TreeRow({
           const stats =
             typeof nodeStats === 'object' &&
             formatTreeStats(nodeStats, row.children === 0);
+          const table = genericTables[treeName] as SpecifyTable<AnyTree>;
+          const [resource] = useLiveState<
+            SpecifyResource<AnySchema> | undefined
+          >(
+            React.useCallback(() => {
+              const table = genericTables[treeName] as SpecifyTable<AnyTree>;
+              const parentNode = new table.Resource({ id: row.nodeId });
+              let node = parentNode;
+              return node;
+            }, [row.nodeId, treeName])
+          );
+          const [loadedResource] = useAsyncState(
```

Comment:
performance of calling this for each row is going to be bad.
tree performance is bad as is.
instead, this information might need to be provided by the back-end, or on-demand, or enabled only for certain ranks (like we do with tree stats right now)
otherwise, this would make a lot of network requests...

cc @realVinayak